source/codegen/common_helpers.py

```diff
@@ -604,3 +604,18 @@ def get_split_attributes_by_type(config):
 
 def supports_raw_attributes(config: dict) -> bool:
     return config.get('supports_raw_attributes', False)
+
+
+def get_enum_value_cpp_type(enum: dict) -> str:
```

Comment:
This method was moved from `service_helpers` (because it is now used in the client) and renamed from `python_to_c` (because that was too low level of a name)